facebook/OpenBIC
# fby35: gl: add usb pid and vid

Summary:
- Configure usb vender id and product id
- Disable spi1_cs1 and spi2_cs1 following the hardware design

Test Plan:
- Check USB port through bic console: pass
- Update BIOS firmware: pass

Test Log:
- Check USB port through bic console root@bmc-oob:/tmp# bic-util slot4 --check_usb_port sb Init libusb Successful!
...
-----------------------------------
Input test data, USB timeout: 3000ms
Received data: e4 09 cc
Check USB port Successful

- Update BIOS firmware root@bmc-oob:~# fw-util slot1 --force --update bios /mnt/data/bios.bin Checking if the server is ready...
This image is not for Yv3.5 platform
...
Strong checksum function is not available, disabling deduplication. Updating BIOS, dedup is off, verification is off.
1024 blocks (1024 written, 0 skipped)...finished.
Elapsed time:  213   sec.
Power-cycling the server...
Force upgrade of slot1 : bios succeeded
